src/Xamarin.Android.Tools.AndroidSdk/Xamarin.Android.Tools.AndroidSdk.nuspec

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>$id$</id>
+    <version>$version$</version>
+    <title>$title$</title>
+    <authors>$author$</authors>
+    <owners>$author$</owners>
+    <licenseUrl>https://github.com/xamarin/xamarin-android-tools/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/xamarin/xamarin-android-tools</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Xamarin tools for interacting with the Android SDK.</description>
+    <copyright>Copyright 2019</copyright>
```

Comment:
Presumably this should be a range, at minimum 2016-2019.  (The first commit in this repo is from 2016.)

Reply:
For that matter, could this be generated somehow?  `git rev-list --max-parents=0 HEAD` or `git log --reverse` can get the first commit, which can provide the initial year.

Reply:
I think we should update this to use the `$copyright$` token, which will automatically use the `AssemblyCopyright` value in `AssemblyInfo.cs` Then we'll update `AssemblyInfo.cs` as needed in the future.

More details: https://docs.microsoft.com/en-us/nuget/reference/nuspec